bin/BuildPackages.sh

```diff
@@ -112,7 +112,7 @@ notice \
 "you require!"
 
 echo_run() {
-  echo "$@"
+  notice "Running $(printf "'%s' " "$@")"
```

Comment:
Why is ````notice "Running" "$@"```` not enough here? Is it because of some escaping issue? 

I am also wondering if this code runs correctly as it is. I would think that there should only be a " before ````Running```` and one after the closing parenthesis (two altogether in the whole line, but definitely not four cut up by the parentheses).